netkan/netkan/cli.py

```diff
@@ -60,7 +60,7 @@ def netkan(debug):
     help='SSH key for accessing repositories',
 )
 def indexer(queue, ckanmeta_remote, token, repo, user, key, timeout):
-    init_ssh(key, '/home/netkan/.ssh')
+    init_ssh(key,  str(Path.home()) + '/.ssh')
     ckan_meta = init_repo(ckanmeta_remote, '/tmp/CKAN-meta')
```

Comment:
`Path` can concatenate things itself and we can pass a `Path` to that function.

```python
init_ssh(key, Path(Path.home(), '.ssh'))
```

Reply:
Now passing Path objects to `init_ssh()`